ddp-workspace/projects/toolkit/src/lib/components/session-expired/session-expired-redesigned.component.ts

```diff
@@ -24,25 +24,15 @@ import { ActivatedRoute } from '@angular/router';
         </main>`
 })
 export class SessionExpiredRedesignedComponent extends SessionExpiredComponent implements OnInit {
```

Comment:
`SessionExpiredRedesignedComponent` is used by TestBoston study, but Angio uses the parent `SessionExpiredComponent`, so I moved things into there.

Reply:
wondering why angio can't use SessionExpiredRedesignedComponent as well. I can see that not only testboston, but atcp, brain and osteo use it as well. What's the diff between those 4 projects and angio?

Reply:
I'm not sure either! (I haven't followed development on frontend too closely.) I think it is because CMI studies (angio, brain, etc) uses different visuals / styles, while studies like TestBoston wants to display things differently (hence the component name "redesigned"). I think it's best to keep whatever design Angio is currently using.